src/core/Internal.ml

```diff
@@ -174,52 +173,49 @@ module Make(Plugin : PLUGIN)
     let[@inline] assignable v = v.v_assignable
     let[@inline] weight v = v.v_weight
     let[@inline] mark v = v.v_fields <- v.v_fields lor seen_var
-    let[@inline] unmark v = v.v_fields <- v.v_fields land (lnot seen_var)
-    let[@inline] marked v = (v.v_fields land seen_var) <> 0
+    let[@inline] unmark v = v.v_fields <- v.v_fields land lnot seen_var
+    let[@inline] marked v = v.v_fields land seen_var <> 0
 
-    let make (st:st) (t:formula) : var * Solver_intf.negated =
+    let make (st : st) (t : formula) : var * Solver_intf.negated =
       let lit, negated = Formula.norm t in
-      try
-        MF.find st.f_map lit, negated
+      try MF.find st.f_map lit, negated
       with Not_found ->
         let cpt_double = st.cpt_mk_var lsl 1 in
-        let rec var  =
+        let rec var =
           { vid = st.cpt_mk_var;
-            pa = pa;
-            na = na;
+            pa;
+            na;
             v_fields = 0;
             v_level = -1;
-            v_idx= -1;
+            v_idx = -1;
             v_weight = 0.;
             v_assignable = None;
-            reason = None;
-          }
+            reason = None }
         and pa =
-          { var = var;
-            lit = lit;
-            watched = Vec.create();
+          { var;
+            lit;
+            watched = Vec.create ();
             neg = na;
             is_true = false;
             aid = cpt_double (* aid = vid*2 *) }
         and na =
-          { var = var;
+          { var;
             lit = Formula.neg lit;
-            watched = Vec.create();
+            watched = Vec.create ();
             neg = pa;
             is_true = false;
-            aid = cpt_double + 1 (* aid = vid*2+1 *) } in
+            aid = cpt_double + 1 (* aid = vid*2+1 *) }
+        in
         MF.add st.f_map lit var;
         st.cpt_mk_var <- st.cpt_mk_var + 1;
         Vec.push st.vars (E_var var);
         var, negated
 
     (* Marking helpers *)
-    let[@inline] clear v =
-      v.v_fields <- 0
+    let[@inline] clear v = v.v_fields <- 0
 
     let[@inline] seen_both v =
-      (seen_pos land v.v_fields <> 0) &&
-      (seen_neg land v.v_fields <> 0)
+      seen_pos land v.v_fields <> 0 && seen_neg land v.v_fields <> 0
```

Comment:
The removal of parentheses (even if they were not semantically needed), makes this much harder to read, :(